src/lib/structures/expr.mli

```diff
@@ -72,7 +72,25 @@ and quantified = private {
   (** The underlying formula. *)
 
   toplevel : bool;
-  (** Determine if the quantified formula is at the top level.
+  (** Determine if the quantified formula is at the top level of an asserted
+      formula.
+
+      An {e asserted formula} is a formula introduced by {e (assert ...)} or
+      generated by a function definition with {e (define-fun ...)}.
```

Comment:
I am not familiar with the formatting `{e ...}` for documentation, not sure what it does. Afaik brackets `[...]` or `[|...|]` are usually used for code.
cf: https://ocamlverse.net/content/documentation_guidelines.html

Reply:
`{e ...}` emphasizes the text. I'm used to use brackets for OCaml code only but I haven't strong opinions on this.

Reply:
I use the bracket syntax in the last commit.